src/Transfer/Resources/Database/Attributes/EmailAttribute.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Utopia\Transfer\Resources\Database\Attributes;
+
+use Utopia\Transfer\Resources\Database\Attribute;
+use Utopia\Transfer\Resources\Database\Collection;
+
+class EmailAttribute extends Attribute
```

Comment:
```suggestion
class Email extends Attribute
```

No need of the Attribute suffix. Make sure to change it in the file name.